rln/src/public.rs

```diff
@@ -58,14 +58,15 @@ impl RLN<'_> {
     ///
     /// Input parameters are
     /// - `tree_height`: the height of the internal Merkle tree
-    /// - `input_data`: a reader for the string path of the resource folder containing the ZK circuit (`rln.wasm`), the proving key (`rln_final.zkey`) or (`rln_final.arkzkey`) and the verification key (`verification_key.json`).
-    ///
+    /// - `input_data`: include next parameters
+    ///     - `resources_folder`: a reader for the string path of the resource folder containing the ZK circuit (`rln.wasm`), the proving key (`rln_final.zkey`) or (`rln_final.arkzkey`) and the verification key (`verification_key.json`).
+    ///     - `tree_config`: a reader for a string containing a json with the merkle tree configuration
     /// Example:
     /// ```
     /// use std::io::Cursor;
     ///
     /// let tree_height = 20;
-    /// let resources = Cursor::new(json!({"resources_folder": "tree_height_20"});
```

Comment:
good catch, should we probably remove this as a config parameter? `resources_folder`

Reply:
hm, I guess if the data from our directory is always be used, then yes